eng/common/scripts/Verify-Links.ps1

```diff
@@ -191,7 +191,7 @@ function CheckLink ([System.Uri]$linkUri)
         if ($null -ne $statusCode) {
           Write-Host "[$statusCode] while requesting $linkUri"
         }
-        else {
+        elseif (!$linkUri.ToString().StartsWith("#")){
```

Comment:
Why is this needed? We never used to have these anchor tags being reported as broken links.

Reply:
We have the exception message get prints out, which does not provide so much information but not host associate with the link. I am thinking to remove the exception message.

Reply:
Does this need to be resolved before merging?

Reply:
The anchor link prints exception messages, which is unnecessary to me. 
![image](https://user-images.githubusercontent.com/48036328/95900098-4287cd00-0d46-11eb-92c9-b84f1db617df.png)

I am thinking we do not print anything for anchor link exception. However, I am flexible if the info is useful to us.

Reply:
OK if simple anchor links are causing exceptions then I also agree we should try to avoid them if we can as that just adds noise. However from a quick check it seems like `$linkUri = [System.Uri]"#test-anchor"` works without throwing an exception so what is throwing this exception?

Reply:
If that is the case we should just avoid the Invoke-WebRequest call completely as it will always fail for these cases.

Reply:
I actually expect that Invoke-WebRequest will fail for any relative url.